init-tools.sh

```diff
@@ -106,10 +113,11 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
             __DOTNET_LOCATION="https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/${__DOTNET_TOOLS_VERSION}/${__DOTNET_PKG}.${__DOTNET_TOOLS_VERSION}.tar.gz"
             # curl has HTTPS CA trust-issues less often than wget, so lets try that first.
             echo "Installing '${__DOTNET_LOCATION}' to '$__DOTNET_PATH/dotnet.tar'" >> $__init_tools_log
-            if command -v curl > /dev/null; then
-                curl --retry 10 -sSL --create-dirs -o $__DOTNET_PATH/dotnet.tar ${__DOTNET_LOCATION}
-            else
+            which curl > /dev/null 2> /dev/null
```

Comment:
Why have you changed to `which`? We have switched to using "command -v" everywhere recently. See https://github.com/dotnet/coreclr/pull/8823